CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
   * Added tvOS/watchOS/macOS support and examples
```

Comment:
Please add a period to end of sentence for consistency with other pull request additions.